onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -1073,6 +1074,208 @@ def aten_rrelu_with_noise_backward(
     raise NotImplementedError()
 
 
+@onnxscript.script()
+def _causal_attention_mask(query: TFloat, key: TFloat) -> TFloat:
+    # attn_mask = torch.ones(L, S, dtype=torch.bool).tril(diagonal=0)
+    # attn_mask = attn_mask.masked_fill(not attn_mask, -float('inf')) if attn_mask.dtype==torch.bool
+    target_length = op.Shape(query)[-2:-1]
+    source_length = op.Shape(key)[-2:-1]
+    # attn_mask = torch.ones(L, S) := {
```

Comment:
For my learning, what is this style of comment?

Reply:
it was my own style :D lmk if they should look differently

Reply:
To show that the lines are equivalent to the torch op in the comment